lib/srv/desktop/rdp/rdpclient/src/rdpdr.rs

```diff
@@ -79,12 +79,13 @@ impl RdpdrBackend for TeleportRdpdrBackend {
         self.scard.handle(req, call)
     }
 
-    fn handle_drive_io_request(&mut self, req: ServerDriveIoRequest) -> PduResult<()> {
+    fn handle_drive_io_request(&mut self, req: ServerDriveIoRequest) -> PduResult<Vec<SvcMessage>> {
```

Comment:
Why do we need to change the return type here?

Reply:
`RdpdrBackend` trait was changed in current version of IronRDP